gensim/viz/poincare.py

```diff
@@ -59,7 +59,7 @@ def poincare_2d_visualization(model, tree, figure_title, num_nodes=50, show_node
         mode='markers',
         marker=dict(color='rgb(30, 100, 200)'),
         text=node_labels,
-        textposition='bottom'
+        textposition='bottom center'
```

Comment:
What's about older plotly version? I think PR changes works only for new. This should be compatible with new & old plotly.

Reply:
Checked - it works for 2.7.0 too, great!

Reply:
Thanks! :)